### DISCUSSION TITLE
MG5050 on OpenHAB please help

### DISCUSSION THREAD

**Discussion by captain1019** [ParadoxAlarmInterface/pai#454] [Q&A]
hi

Wondering if anyone can assist with this:
https://github.com/ParadoxAlarmInterface/pai/issues/438

I see that the official Binding Integration for OpenHAB is not maintained by ParadoxAlarmInterface/pai

I just want to know is it still possible to get the MG5050 working nevertheless? Im a bit new to this all.
I saw this documentation https://github.com/ParadoxAlarmInterface/pai/wiki/OpenHAB
Does this setup work for MG5050? If yes, how do i bind the Bridge manually in things/broker.things if i use MQTT Broker via the UI in OpenHAB? I heard that you cant mix the 2 UI and manually coding it.

thanks in advance

**Comment by yozik04** [comment]
ParadoxAlarmInterface/pai works with MG5050. You can configure OpenHAB to listen to certain topics on MQTT where PAI will publish. I previously had setup with OpenHAB and it worked. I switched to HomeAssistant some years ago. You need to investigate how to configure OpenHAB MQTT things with minimal effort.
